### ISSUE TITLE
Always have to use --clean

### ISSUE THREAD

**Issue by fearoffish** [applicationsonline/librarian#45]
Maybe I'm not using librarian correctly, but running this never updates my cookbooks:

```
librarian-chef update cookbook_name
```

I have to do:

```
librarian-chef install --clean
```

Then it updates from my github repository.  Am I doing something wrong?

**Comment by fearoffish** [comment]
I should mention I'm using the 0.0.14 gem.

**Comment by fearoffish** [comment]
It also fails to work for a :path'ed repository.  No one else has this problem?

**Comment by yfeldblum** [comment]
What is `librarian-chef` failing to do?
- Rewrite `Cheffile.lock`
- Install updated set of cookbooks into `cookbooks/`

Does this happen with cookbooks from only `:git` and `:path` sources, or also with cookbooks from `:site` sources?

**Comment by fearoffish** [comment]
Apparently you have to run install after update, silly me!

```
librarian-chef update my_cookbook
librarian-chef install
```

Closing, apologies.

**Comment by yfeldblum** [comment]
Addressed by #46.
